index.html

```diff
@@ -260,5 +261,56 @@ <h2 class="icons-header">LET'S CONNECT</h2>
         />
       </div>
     </section>
+    <section class="contact-page-container">
```

Comment:
- [ ] Your stylesheet is truly not displaying as specified by the previous reviewer. YOu should add a dot before the relative path of your style URL to display the styles 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/220445074-9225f47c-d989-4590-ba65-91d7318cbdc9.png)